rpc/types_transaction_response.go

```diff
@@ -18,3 +18,30 @@ type AddDeployAccountTransactionResponse struct {
 type AddInvokeTransactionResponse struct {
 	TransactionHash *felt.Felt `json:"transaction_hash"`
 }
+
+type TransactionResponse struct {
+	TransactionHash *felt.Felt `json:"transaction_hash"`
+	ClassHash       *felt.Felt `json:"class_hash"`
+	ContractAddress *felt.Felt `json:"contract_address"`
```

Comment:
Let's add the `omitempty` tag to the ClassHash and ContractAddress fields, that way we don't need the "ConvertToTransactionResponse" function nor the AddInvokeTransactionResponse, AddDeployAccountTransactionResponse, and the AddInvokeTransactionResponse types; we can remove them.

Reply:
done